flybywiresim/a32nx
# Improved fix for uncontrollable dive

Improves upon the previous PR #2698 

## Summary of Changes

- Improved notice of Flaps auto-retraction from config 1 + F to config 1 in Upper ECAM and Core Speeds
- Enabled auto-retraction in config for drag, which also enables the flap retraction in the eternal view

Allows the auto-retraction of 1+F to 1 for drag, but not for lift. This enables the animation in the external model to show the automatic flap retraction. So, now we can have the visual flap retraction (and the drag reduction associated with it), but retains the lift of 1+F just like the preliminary fix. The external view will show the flap retract above 210 knots, but it will not lead to the unrecoverable stall condition.
## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): donbikes#4084

## Testing instructions

This should be tested in a similar manner as the testing for PR #2698, except add a check of the external view that the flap retracts at speeds above 210 knots, and of the internal view that the flap symbol also shows the flap retraction.

1. Start on the runway (any runway should do) at the default weight.
2. Go to external view. No need to be distracted by any instruments, warnings,, CDU setup, or anything else. 
3. Release parking brakes, go to TOGA thrust, rotate around 130 knots (not critical), and retract the landing gear with positive rate of climb.
4. Just keep a gentle climb with the primary purpose of allowing the airplane to accelerate. Do not retract the flap.
5. When accelerating through 210 knots, confirm that the flap auto-retracts. (It takes a little while, and depending on how fast you are accelerating it may not appear to start right away.)
6. Switch to internal view and confirm that the ECAM flap symbol shows that the flap has retracted (from 1+F to 1) and that VFE on the PFD speed tape has increased from 215 knots to 230 knots.
7. Reduce speed to below 210 knots and confirm that the flap re-deploys in the external view, and that the ECAM symbol changes back to 1+F, and VFE reduces again to 215 knots.
8. Increase speed again, and either disable the overspeed aural warning by pushing the master caution button or switch back to external view and continue to accelerate out to 280-290 knots to verify that this PR does not cause the unrecoverable stall issue to reappear.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
